spec/features/families/admin_views_all_active_families_spec.rb

```diff
@@ -0,0 +1,15 @@
+require "rails_helper"
+
+feature "Admin can see all the active families when in families#index" do
+  scenario "successfully", js: true do
+    create(:all_true_family)
+    create(:all_false_family)
+
+    visit families_path
+    find("#active_families_button").click
+
+    expect(page).to have_text(I18n.t("families.index.buttons.active"))
+    expect(page).to have_text(true)
+    expect(page).not_to have_text(false)
```

Comment:
A suggestion to change the feature spec to be more specific as to what we see on the page: i.e.
```
active_family = create(:all_true_family, last_name: "Russo")
inactive_family = create(:all_false_family, last_name: "De Luca")

visit families_path
 
expect(page).to have_text("Russo")
expect(page).to have_text("De Luca")
```